internetarchive/openlibrary
# Enable title field and disable author field

Closes #3523 



### Technical

Title field is enabled and author field is read-only in the add another edition page
### Testing

1.  Go to one edition/book record
2. Click to add a new edition of this work

Expected: Title field should be enabled and author field should be read-only in the add another edition page
### Testing
### Screenshot

![edit2](https://user-images.githubusercontent.com/50094420/103104616-27a2c600-45dd-11eb-9488-89837d1b85fd.JPG)
![edit1](https://user-images.githubusercontent.com/50094420/103104620-2d001080-45dd-11eb-9627-a77b8ad79027.JPG)

### Stakeholders

@mekarpeles @tabshaikh
